sdk/src/main/java/com/podio/sdk/domain/Label.java

```diff
@@ -0,0 +1,22 @@
+package com.podio.sdk.domain;
+
+/**
+ * Created by ianhenry on 6/28/16.
+ */
+public class Label {
+    private final Long label_id = null;
+    private final String text = null;
+    private final String color = null;
```

Comment:
Would it make sense to have color as some sort of an enumeration instead of a string?